server/backend/users/tests/test_serializers_auth.py

```
@@ -0,0 +1,84 @@
+import pytest
+from rest_framework.exceptions import ValidationError
+from rest_framework.test import APIClient
```

Ruff (0.8.2)

`rest_framework.test.APIClient` imported but unused

Remove unused import: `rest_framework.test.APIClient`

(F401)